Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -74,7 +74,7 @@ struct SelectTracks {
       int status_prong = 3; // selection flag , 2 bits on
 
       auto trackPt = track.pt();
-      if (b_dovalplots) {
```

Comment:
@nzardosh here are the changes I propose in view of a better alternative, what do you think?